src/category_theory/limits/creates.lean

```diff
@@ -122,6 +122,10 @@ lemma has_limit_of_created (K : J ⥤ C) (F : C ⥤ D)
   [has_limit (K ⋙ F)] [creates_limit K F] : has_limit K :=
 has_limit.mk { cone := lift_limit (limit.is_limit (K ⋙ F)),
   is_limit := lifted_limit_is_limit _ }
+  
+/-- If `F` creates limits, and `D` has all limits, then `C` has all limits. -/
+def has_limits_of_has_limits_creates_limits (F : C ⥤ D) [has_limits D] [creates_limits F] :
+  has_limits C := ⟨λ J I, by letI := I; exact ⟨λ G, has_limit_of_created G F⟩⟩
```

Comment:
Just in case you weren't aware, there's a slightly shorter way:
```suggestion
  has_limits C := ⟨λ J I, by exactI ⟨λ G, has_limit_of_created G F⟩⟩
```

Reply:
Thanks! Yes, I'll use this.